apps/backoffice-v2/src/common/components/organisms/DataTable/DataTable.tsx

```diff
@@ -104,7 +104,7 @@ export const DataTable = <TData extends RowData, TValue = any>({
         const newSortingState = sortingUpdaterOrValue(prevSortingState);
 
         onSort({
-          sortBy: newSortingState[0]?.id || sortBy,
+          sortBy: newSortingState[0]?.id?.replace(/_/g, '.') || sortBy,
```

Comment:
Tanstack table converts columns that are period separated to snake_case. In the URL we want to represent objects so we transform back to a period.